bin/helm_deploy

```diff
@@ -18,7 +18,7 @@ fi
 release_name="${1}"
 namespace="${2}"
 
-CHART_VERSION="${CHART_VERSION:-3.5.1}"
+CHART_VERSION="${CHART_VERSION:-3.7.0}"
```

Comment:
I tried a deploy. it failes 

<img width="573" height="128" alt="Image" src="https://github.com/user-attachments/assets/ae3d3f8b-0115-4529-bc9b-f91251149f17" />

should we try 3.6.0 instead as that's the highest version listed in Hyrax [packages](https://github.com/orgs/samvera/packages/container/package/charts%2Fhyrax)?